assets/models/system/sshleifer-distilbart-cnn-12-6/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/sshleifer-distilbart-cnn-12-6/a4f8f3e/mlflow_model_folder
+  container_path: huggingface/sshleifer-distilbart-cnn-12-6/1703564895/mlflow_model_folder
   storage_name: automlcesdkdataresources
```

Comment:
These models have Whisper packages, please refer to other models and remove them if not required.

Reply:
Hi Bhavana, now Updated Please check Thanks.